client/data/launchpad-navigator/use-launchpad-navigator.ts

```diff
@@ -0,0 +1,50 @@
+import { useQuery } from '@tanstack/react-query';
+import apiFetch, { APIFetchOptions } from '@wordpress/api-fetch';
```

Comment:
Should this not be `{ type APIFetchOptions }`?
```suggestion
import apiFetch, { type APIFetchOptions } from '@wordpress/api-fetch';
```